### ISSUE TITLE
Right clicking a Rem in Graph View opens the Rem in a New Pane

### ISSUE THREAD

**Issue by Remember447** [remnoteio/remnote-issues#572]
**Describe the bug**
When a Rem is right clicked in graph view, the Rem is opened in a New Pane.  This causes problems when one wants to select an option in the right click menu.

**To Reproduce**
1. Open RemNote
2. Select any document
3. Open graph view for the document
4. Right click on a Rem 
5. See error

**Expected behavior**
I would expect that *left* clicking on a Rem would open the Rem in a new pane and that *right* clicking wouldn't.  This would allow one to select something from the right click menu when a Rem is right clicked.  If the new pane opened, then the right click menu is often blocked (or partially blocked) by the new pane.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Version 1.2

**Comment by mfranzs** [comment]
Thanks for reaching out. I'm having trouble replicating this - are you still experiencing it?

**Comment by Remember447** [comment]
Yes, I am still experiencing it.  I do think that I could be more precise with my vocabulary though.  When I right click on a *node* in the graph view, then the node's corresponding Rem opens up in another pane.  I attached a video of the right click opening a Rem in a new pane.  In the video, the yellow marks signify a left click and the blue marks signify a right click.

https://user-images.githubusercontent.com/77589970/105194757-ec29e780-5b07-11eb-97dd-4ef17344acab.mp4

**Comment by drkvj** [comment]
I have tried to reproduce this on the Mac app, but everything seems to be working fine from my end. This could be a Windows issue, but @mfranzs should've been able to catch this in that case. Need to explore this more and see if other users are experiencing this as well.
